### PR TITLE
Sgproj 1461 Module upgrade

### PR DESCRIPTION
**Pull request by DJKaswa**
## Change description

> Upgrades module's provider to 4.22.0 and sets it to be backward compatible with older versions since some projects are specifically locked on older versions.

## Type of change
- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

> Fix [#1]() 

### Development

- [x] Lint rules pass locally
- [x] Application changes have been tested thoroughly
- [x] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [x] Pull request linked to task tracker where applicable

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.75"
+      version = "<= 4.9"
```
Comment (rchildress87) [inline versions.tf:7]: Why are we arbitrarily updating the AWS provider's version constraint? The last breaking changes I see that involved any of the resources  in this module were in the 3.0.0 release of the AWS provider. I'd either leave as-is or set to `>= 3.0.0`. An issue can be reported if there is a breaking change or vulnerability discovered in the future. @wesleykirklandsg ?

Comment (DJKaswa) [reply versions.tf:7]: @rchildress87 I am confused on your comment here. I thought we wanted to upgrade providers as well, part of module upgrade initiative? Am I wrong?

Comment (wesleykirklandsg) [reply versions.tf:7]: @DJKaswa We're never going to force a below version, if there is a major breaking chang we will do a breaking change in the module release. Undo the change as requested, please.

Comment (DJKaswa) [reply versions.tf]: @rchildress87 -- I've changed the provider as per our conversation.

**Comment by DJKaswa** [comment]
closing this branch as no upgrade will be done on the provider.

**Pull request closed without merging**